src/services/infra/DynamoDBClient.ts

```diff
@@ -82,38 +78,13 @@ export default class DynamoDBDocClient {
     )
   }
 
-  getItem = async (
-    tableName: string,
-    key: string
-  ): Promise<GetCommandOutput> => {
+  getAllItems = async (tableName: string): Promise<ScanCommandOutput> => {
     const params = {
       TableName: tableName,
-      Key: { appId: key },
-    }
-
-    return this.withLogger(
-      this.dynamoDBDocClient.send(new GetCommand(params)),
-      "get"
-    )
-  }
-
-  updateItem = async ({
-    TableName,
-    Key,
-    UpdateExpression,
-    ExpressionAttributeNames,
-    ExpressionAttributeValues,
-  }: UpdateParams): Promise<UpdateCommandOutput> => {
-    const params: UpdateCommandInput = {
-      TableName,
-      Key,
-      UpdateExpression,
-      ExpressionAttributeNames,
-      ExpressionAttributeValues,
     }
     return this.withLogger(
-      this.dynamoDBDocClient.send(new UpdateCommand(params)),
-      "update"
+      this.dynamoDBDocClient.send(new ScanCommand(params)),
+      "scan"
```

Comment:
hmm why are we using scan? Is this is DDB Scan command? I think we should not use scan as much as possible

https://dynobase.dev/dynamodb-scan/

Reply:
Hey! Yup, we shouldn't use scan as much as possible due to it being slow. However, in our use case, we
1. Use DynamoDB to store state of launches during the launch process, prior to being updated in our RDS DB 
2. The amount of entries is small (on our biggest launch dates we only had ~30 launches in a day), which translates to 30 rows

The alternative to this if we dont use the `scan` command is that we store the key in our RDS-DB, which will involve a DB migration to have this functionality, which feels over-engineered for our particular use case! 

As such, I think we should stick scanning our entire DB, wdty?

Reply:
given is < 100 rows, I think it's fine for this use case then